roles/etcd/defaults/main.yaml

```diff
@@ -2,11 +2,9 @@
 r_etcd_common_backup_tag: ''
 r_etcd_common_backup_sufix_name: ''
 
-l_etcd_bootstrapped: '{{ openshift.node.bootstrapped }}'
-
 # TODO(michaelgugino): Remove these in 3.11
 l_is_etcd_system_container: "{{ (openshift_use_etcd_system_container | default(openshift_use_system_containers | default(false)) | bool) }}"
-l_etcd_static_pod: "{{ not (r_etcd_common_skip_command_shim is defined and r_etcd_common_skip_command_shim) or l_etcd_bootstrapped }}"
+l_etcd_static_pod: "{{ openshift.node.bootstrapped }}"
```

Comment:
Is r_etcd_common_skip_command_shim unused?

Reply:
The only other reference has been removed.  It was being used to handle 3.9 to 3.10 upgrades when not all nodes were bootstrapped.  Unless I've got it all wrong.

Reply:
I'm unfamiliar with this code.  In any case, if it's not required to fix what we're doing here, let's not touch it in 3.10.  If it's required, lgtm, let me know.

Reply:
The problem I am fixing here is that `r_etcd_common_etcd_runtime` was not being set correctly and therefore the wrong command was being used for stand-alone hosts.  This issue came up in the bug once scaleup was fixed when running with static pods.  If trying to scale up stand-alone hosts the wrong command was being used.